<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompressDialog</class>
 <widget class="QDialog" name="CompressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compress</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>414</y>
     <width>174</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxOffset">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>89</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset for drift indexing</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInputFile">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutputFile">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInput">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutput">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>129</y>
     <width>255</width>
     <height>268</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxCorrectors">
      <property name="text">
       <string>Remove Correctors</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxSextupoles">
      <property name="text">
       <string>Remove Sextupoles</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxSolenoids">
      <property name="text">
       <string>Remove Solenoids</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxInstruments">
      <property name="text">
       <string>Remove Instruments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxRemoveApertures">
      <property name="text">
       <string>Remove Apertures</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxRemoveWField">
      <property name="text">
       <string>Remove Wakefield Elements</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxRemoveScatter">
      <property name="text">
       <string>Remove Scattering Elements</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxRemoveVars">
      <property name="text">
       <string>Remove variables from element definitions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CompressDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CompressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonInput</sender>
   <signal>clicked()</signal>
   <receiver>CompressDialog</receiver>
   <slot>selectInputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutput</sender>
   <signal>clicked()</signal>
   <receiver>CompressDialog</receiver>
   <slot>selectOutputFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectInputFile()</slot>
  <slot>selectOutputFile()</slot>
 </slots>
</ui>
